tests/index.test.js

```diff
@@ -0,0 +1,23 @@
+import { map, filter } from '../index.js';
+
+describe('map', () => {
+  it('replaces .map with a .length function', () => {
```

Comment:
I'm thinking you might want to describe this test a little more clearly? I'm not quite sure what this description refers to.